### DISCUSSION TITLE
Example Retype-powered Sites

### DISCUSSION THREAD

**Discussion by DannyBen** [retypeapp/retype#106] [Show and tell]
Hello everybody,

I have built some documentation sites using Retype.

One thing that helped and inspired me, was the fact that I could look at retype.com for inspiration, and to peak at its code to quickly copy a nice widget code or other snippet from a "real world" example.

So for this reason (and honestly, also to show off a little...), I am posting the Retype-powered sites that I migrated from another documentation tool, or from a long README.

- [Runfile documentation](https://runfile.dannyb.co) - this is the smallest of the bunch.
- [Bashly documentation](https://bashly.dannyb.co) - this is the first I have done (built from scratch, not migrated).
- [Jobly documentation](https://jobly.dannyb.co) - this is the largest one.
- [Victor documentation](https://victor.dannyb.co/) - the latest one, medium size.

Hoping these can serve as another source of inspiration and reference to future Retype users.

**Comment by geoffreymcgill** [comment]
Your sites look amazing! Very nice documentation. Excellent logos too. So many software projects don't pay attention to that detail. A nice logo brings a polished professional look.

I'm going pull some component configs from your sites into the retype.com docs. I want to figure out somewhere to use the button component linking to the demo|sample. I like the use of `icon: dot` for the sub-pages within a folder too. Super clean. 

Nice job.
